test/specs/configs/empty.json

```diff
@@ -0,0 +1,25 @@
+{
+    "strains": {
+        "default": {
+            "code": "",
+            "condition": "",
+            "content": {
+                "filepath": "",
```

Comment:
Why `filepath` instead of `path`?

Reply:
this is how it it defined in `git-url-parse`. but I think `path` or `root` might be better.